apps/www/_blog/2023-03-01-geo-queries-with-postgis-in-ionic-angular.mdx

```diff
@@ -127,21 +127,21 @@ Those are some powerful calculations, and we can easily use them through the Pos
 ```sql
 -- create database function to find nearby stores
 create or replace function nearby_stores(lat float, long float)
-returns setof record
+returns table (id public.stores.id%TYPE, name public.stores.name%TYPE, description public.stores.description%TYPE, lat float, long float, dist_meters float)
```

Comment:
The update in this article is the same as the PostGis guide, and changes the return type from `setof record` to `table`.